### PR TITLE
Insurance context

### PR DESCRIPTION
**Pull request by RobinCsl**

### CODE REVIEW

File: app/MMB/src/scenes/tripServices/insurance/insuranceOverviewScene/InsuranceOverviewScene.js [hunk 1]
```diff
@@ -3,26 +3,65 @@
...
+  +passengers: Passenger[],
+  +changes: Change[],
 |};
 
 class InsuranceOverviewScene extends React.Component<Props> {
+  componentDidMount() {
+    const { data } = this.props;
+    const passengers = data.singleBooking.passengers;
+    const insurancePrices = data.singleBooking.insurancePrices;
+    // console.log(this.props);
```
Comment (tbergq) [inline app/MMB/src/scenes/tripServices/insurance/insuranceOverviewScene/InsuranceOverviewScene.js]: can you delete this comment

Comment (RobinCsl) [reply app/MMB/src/scenes/tripServices/insurance/insuranceOverviewScene/InsuranceOverviewScene.js]: Good catch! Thanks

File: app/MMB/src/scenes/tripServices/insurance/insuranceOverviewScene/InsuranceOverviewSceneContainer.js [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+import * as React from 'react';
+import { graphql, PublicApiRenderer } from '@kiwicom/mobile-relay';
+import { type NavigationType } from '@kiwicom/mobile-navigation';
+
+import BookingDetailContext from '../../../../context/BookingDetailContext';
+import type { InsuranceOverviewSceneQueryResponse } from './__generated__/InsuranceOverviewSceneContainerQuery.graphql';
+import InsuranceOverviewScene from './InsuranceOverviewScene';
+
+type ContainerProps = {|
+  navigation: NavigationType,
```
Comment (tbergq) [inline app/MMB/src/scenes/tripServices/insurance/insuranceOverviewScene/InsuranceOverviewSceneContainer.js]: should be read only

File: app/MMB/src/scenes/tripServices/insurance/insuranceOverviewScene/InsuranceOverviewSceneContext.js [hunk 3]
```diff
@@ -0,0 +1,147 @@
...
+    ];
+    this.setState({ passengers: updatedPassengers, changes: updatedChanges });
+  };
+
+  render = () => (
+    <ContextProvider value={this.state}>{this.props.children}</ContextProvider>
+  );
+}
+
+export default { Consumer, Provider };
```
Comment (tbergq) [inline app/MMB/src/scenes/tripServices/insurance/insuranceOverviewScene/InsuranceOverviewSceneContext.js]: Could you add some tests for this context?

File: app/MMB/src/scenes/tripServices/insurance/insuranceOverviewScene/__tests__/InsuranceOverviewSceneContext.test.js [hunk 4]
```diff
@@ -0,0 +1,138 @@
...
+      {
+        databaseId: 123,
+        from: 'NONE',
+        to: 'NONE',
+      },
+    ]);
+    expect(instance.props.initialised).toBe(true);
+  });
+
+  it("updates a passenger's insurance", () => {
```
Comment (tbergq) [inline app/MMB/src/scenes/tripServices/insurance/insuranceOverviewScene/__tests__/InsuranceOverviewSceneContext.test.js]: why are there double coutes here?

Comment (tbergq) [reply app/MMB/src/scenes/tripServices/insurance/insuranceOverviewScene/__tests__/InsuranceOverviewSceneContext.test.js]: oh, right, it is because of `'`

Comment (RobinCsl) [reply app/MMB/src/scenes/tripServices/insurance/insuranceOverviewScene/__tests__/InsuranceOverviewSceneContext.test.js]: #prettier

**Pull request merged**